modules/replication/dockerimage-replication/modulestack.yaml

```diff
@@ -45,13 +45,13 @@ Resources:
               - "s3:ListBucket"
               - "s3:PutObject"
             Effect: Allow
-            Resource: "arn:aws:s3:::*-dockerimages-replications-*"
+            Resource: "arn:aws:s3:::*-dkr-img-rep-*"
           - Action:
               - "s3:CreateBucket"
               - "s3:ListBucket"
               - "s3:PutObject"
             Effect: Allow
-            Resource: "arn:aws:s3:::*-dockerimages-replications-*/*"
```

Comment:
Can't do this....you need to support backward compatibility, so both versions of the bucket name need to be in the policy, else users will be hosed..